test-network-function/networking/suite.go

```diff
@@ -188,23 +194,52 @@ func processContainerIpsPerNet(containerID *configsections.ContainerIdentifier,
 	netsUnderTest[netKey] = entry
 }
 
+func IsIPv4(address string) bool {
```

Comment:
are you sure the input string is  IP ? otherwise, you can use net package, it has utilities to parse IP and validate it

Reply:
fixed